packages/main/src/themes/Table.css

```diff
@@ -27,4 +27,16 @@ ui5-table {
 
 .sapUiSizeCompact .sapWCTableHeader {
 	height: 2rem;
+}
+
+.sapWCTableNoData{
+	overflow: hidden;
+	padding: .25rem .0625rem;
```

Comment:
I can see you used the same paddings as for one table cell, but here the paddings should be  ```padding: 0.5rem 1rem;```